applicationconfig/client/src/main/java/com/azure/applicationconfig/ConfigurationClient.java

```diff
@@ -161,6 +171,8 @@ public static ConfigurationClientBuilder builder() {
      *
      * <p><strong>Code Samples</strong></p>
      *
+     * <p>Update a setting with the key "prodDBConnection" to have the value "updated_db_connection".</p>
```

Comment:
This doesn't match what is shown in the code below. Firstly, there is a typo in the code `prodDCConnection`, and secondly, it says `db_connection` rather than `updated_db_connection`